fibsem/microscope.py

```diff
@@ -2562,10 +2564,16 @@ def draw_bitmap_pattern(self, pattern_settings: FibsemBitmapSettings):
 
         # Get bitmap from pattern settings
         bitmap_pattern = BitmapPatternDefinition()
-        bitmap_pattern.points = pattern_settings.bitmap
+
+        if pattern_settings.interpolate is not None:
```

Comment:
You probably want to also explicitly set the application file to "Si" so that the bitmap and rectangle are drawn with the same application file. I'd do it before this interpolate function is called. Have a look at the draw_circle func for example. 
It "should" be already set via setup milling, but not guaranteed.

Reply:
I don't understand this, why does it have to be Si? I don't understand why the default application file is overridden for circle in that way. Given that `self._default_application_file` defaults to "Si" but would be updated by `setup_milling` (so will always be set), wouldn't it be better manually set the default application file to `self._default_application_file`?

Looking into this, I've noticed a few potential issues with how the application file is set, so I'll make a PR based on what I think makes sense (I might be completely wrong but it should be clearer to explain).

Reply:
Actually, I won't mess with a PR now but I'll create an issue.

Reply:
It might not have to be "Si" but it should be a application file that supports Bitmap and Rectangle, otherwise it will throw an error. And if you use a different application file for the Rectangle than Bitmap, it might have different parameters (overlap, pitch etc)

Reply:
I'll hold off on making a change until we've discussed #50 a bit more, since the wrapper I'm suggesting would be able to handle this using something like "Si" as a fallback. If `set_default_application_file` is called at the beginning of the `draw_bitmap` function, it should apply to both automatically, right? Or are there any issues there? I don't entirely trust the API to work as it says.